routers/repo/view.go

```diff
@@ -238,3 +239,148 @@ func Home(ctx *middleware.Context) {
 	ctx.Data["BranchLink"] = branchLink
 	ctx.HTML(200, HOME)
 }
+
+func Language(ext string) string {
```

Comment:
Does prismjs has something like auto-match language? So we don't have to rely on the extension?